### PR TITLE
runtime: add make target for tests

### PR DESCRIPTION
**Pull request by samuelkarp**
*Description of changes:*

New Makefile target, new buildkite step

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -31,3 +31,8 @@ steps:
     commands:
       - "cd snapshotter"
       - "sudo make test EXTRAGOARGS='-v -count=1'"
+
+  - label: ':running_shirt_with_sash: runtime tests'
+    commands:
+      - "cd runtime"
```
Comment (mxpv) [inline .buildkite/pipeline.yml]: Why not to run all tests from root folder? So we don't need to update this if/when we add any other packages with tests (for example `agent` is going to have tests eventually)

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (mxpv) [inline .buildkite/pipeline.yml]: It looks like this is not forced by linter :(

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (samuelkarp) [reply .buildkite/pipeline.yml]: Summarizing our offline discussion: separate steps in Buildkite run in parallel, helping to increase utilization of our build fleet and decrease the test execution time.  Separate steps also gives granular status on the Buildkite status page, so it's easier to dive in and find where a test is failing.

File: .buildkite/pipeline.yml (hunk 2 above)
Comment (samuelkarp) [reply .buildkite/pipeline.yml]: Yeah :slightly_frowning_face: I don't know how that linter works.

**Pull request merged**